docs/build/developer-reference/l1-l2-interop.md

```diff
@@ -25,6 +25,14 @@ L1 to L2 communication is governed by the [`IZkSync.sol`](https://github.com/mat
 - If you prefer to learn-by-doing, the [cross chain governance tutorial](../tutorials/smart-contract-development/cross-chain-tutorial.md) is a practical example of layer interoperability.
   :::
 
+#### L1 to L2 transaction filtering
+
+Hyperchain operator can filter L1→L2 transactions coming through the Diamond proxy. It is achieved by having an address
+of the special contract called TransactionFilterer in the state transition storage. If the filterer exists, it is being
+called in the Mailbox facet with the tx details, and has to return whether the transaction can be executed or not. If
+you want to use this feature, you have to deploy the contract that implements `ITransactionFilterer` interface and set
```

Comment:
nit: `you have to deploy the contract that implements `ITransactionFilterer`` -> `you have to deploy a contract that implements the `ITransactionFilterer``

Reply:
done